src/transaction_signature_options/sign_with_ledger/mod.rs

```diff
@@ -44,6 +46,76 @@ pub struct SignLedgerContext {
 }
 
 impl SignLedgerContext {
+    fn input_blind_agree() -> color_eyre::eyre::Result<bool> {
+        let options: Vec<&str> = vec!["Yes", "No"];
+
+        return Ok(
+            Select::new("Do you agree to continue with blind signature? ", options)
+                .prompt()
+                .map(|selected| selected == "Yes")?,
+        );
+    }
+
+    fn blind_sign_subflow(
+        hash: OnlyBlindSigning,
+        hd_path: BIP32Path,
+        unsigned_transaction: near_primitives::transaction::Transaction,
+    ) -> color_eyre::eyre::Result<near_crypto::Signature> {
+        eprintln!("\n\nBuffer overflow on Ledger device occured. Transaction is too large for normal signature.");
+        eprintln!("\nThe following is SHA256 of unsigned transaction:");
+        eprintln!("{}", hex::encode(&hash.0));
+
+        eprintln!(
+            "\nUnsigned transaction (serialized as base64):\n{}\n",
+            crate::types::transaction::TransactionAsBase64::from(unsigned_transaction)
+        );
+        eprintln!("Before proceeding with blind signature,");
+        eprintln!("you have ability to verify unsigned transaction's details and exact SHA256 correspondence,");
+        eprintln!("if you have concerns with trust to current computer, where near CLI command is being run,");
+        eprintln!("on another computer(s) with the following helper command on near CLI:");
```

Comment:
It seems that part of the sentence is missing. Let's also print the whole sentence in a single print, so it is up to the terminal to wrap it over the lines.

Reply:
Blind signature means that transaction is prepared by CLI, but cannot be reviewed on the Ledger device. In order to be absolutely sure that the transaction you are signing is not forged, take the constructed transaction, verify its content using NEAR CLI on another host or use any other tool capable displaying unsigned NEAR transactions, and confirm that the SHA256 hash matches the one displayed above and another identical one, that will be displayed on your Ledger device after confirming the prompt. Following helper command on near CLI can be used: